R/DBConnection.R

```diff
@@ -106,6 +106,41 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' @export
+dbBreak <- structure(list(), class = "dbiAbort")
```

Comment:
This probably should be a "proper" condition object with `class = c("dbiAbort", "error", "condition")`, see `dput(simpleError("message"))` . Also, I guess dbBreak() should be a function that raises such a condition.